test/relations.test.js

```diff
@@ -835,24 +908,6 @@ describe('relations', function() {
         });
       });
 
-      context('findById with include filter that contains string fields', function() {
```

Comment:
Why was this removed? Please bring it back, use `bdd.contextIf` to skip these tests for Cassandra (if needed).